docs/docs/cloud/cloud-cli.mdx

```diff
@@ -219,6 +229,14 @@ meltano cloud project list
 ╰───────────┴───────────────────────────────┴──────────────────────────────────────────────────────────╯
 ```
 
+The `create` subcommand creates a new Meltano Cloud project.
+
+```sh
+meltano-cloud project create --name example-project --repo-url https://github.com/meltano/squared.git --root-path "data/"
+⠋ Creating project - this may take several minutes...
+Project 'example-project' created successfully.
```

Comment:
Do we need a comment in here about granting GitHub app access?

Reply:
@tayloramurphy what do you think about just linking to the onboarding docs here?